launchable/test_runners/maven.py

```diff
@@ -1,22 +1,47 @@
 import os
+from typing import List
+from xmlrpc.client import Boolean
 import click
 from . import launchable
 
 
-@click.argument('source_roots', required=True, nargs=-1)
+@click.option('--from-file', 'from_files', required=False, multiple=True, type=click.Path(exists=True))
+@click.argument('source_roots', required=False, nargs=-1)
 @launchable.subset
-def subset(client, source_roots):
+def subset(client, source_roots, from_files):
+
+    def is_file(f: str) -> Boolean:
+        return (f.endswith('.java') or f.endswith(".scala") or f.endswith(".kt"))
+
+    def file2class_test_path(f: str) -> List:
```

Comment:
[must]
returning type hinting to `List[Dict[str, str]]`.

Reply:
done https://github.com/launchableinc/cli/pull/410/commits/ef493ee7ae2b89bba0159d1d92f02157964a45de